### PR TITLE
setup devise for login, log out

### PR DESCRIPTION
**Pull request by PhamVanTung**

### CODE REVIEW

File: app/views/devise/confirmations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: {method: :post}) do |f| %>
```
Comment (ngocvnn) [inline app/views/devise/confirmations/new.html.erb]: remove ()

File: app/views/devise/passwords/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<h2>Change your password</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: {method: :put}) do |f| %>
```
Comment (ngocvnn) [inline app/views/devise/passwords/edit.html.erb]: remove ()

File: app/views/devise/passwords/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: {method: :post}) do |f| %>
```
Comment (ngocvnn) [inline app/views/devise/passwords/new.html.erb]: remove()

File: app/views/devise/registrations/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<h1>Edit <%= resource_name.to_s.humanize %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: {method: :put}) do |f| %>
```
Comment (ngocvnn) [inline app/views/devise/registrations/edit.html.erb]: remove()

File: app/views/devise/registrations/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<h1>Sign up</h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
```
Comment (ngocvnn) [inline app/views/devise/registrations/new.html.erb]: remove()

File: app/views/devise/sessions/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,31 @@
+<h1>Log in</h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```
Comment (ngocvnn) [inline app/views/devise/sessions/new.html.erb]: remove()

File: app/views/devise/unlocks/new.html.erb [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend unlock instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: unlock_path(resource_name), html: {method: :post}) do |f| %>
```
Comment (ngocvnn) [inline app/views/devise/unlocks/new.html.erb]: remove()

File: app/views/layouts/_header.html.erb [hunk 8]
```diff
@@ -3,10 +3,26 @@
...
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+              Account <b class="caret"></b>
+            </a>
+            <ul class="dropdown-menu">
+              <li><%= link_to "Profile", '#' %></li>
+              <li><%= link_to "Settings", edit_user_registration_path %></li>
+              <li class="divider"></li>
+              <li>
+                <%= link_to "Log out", destroy_user_session_path, :method=>'delete' %>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: coding convention

File: app/views/layouts/_header.html.erb [hunk 9]
```diff
@@ -3,10 +3,26 @@
...
+            </ul>
+          </li>
+        <% else %>
+          <li><%= link_to "Log in", new_user_session_path %></li>
+        <% end %>
       </ul>
     </nav>
   </div>
-</header>
+</header>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: add new line at end of file

File: app/views/devise/registrations/new.html.erb [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+      <div class="field">
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, class: 'form-control' %>
+      </div>
+
+      <div class="field">
+        <%= f.label :password %>
+        <% if @validatable %>
+        <em>(<%= @minimum_password_length %> characters minimum)</em>
```
Comment (dieunb) [inline app/views/devise/registrations/new.html.erb]: indent

**Pull request merged**